internal/storage/db/internal/model.go

```diff
@@ -21,8 +22,49 @@ const (
 	PolicyDepTbl            = "policy_dependency"
 	PolicyDepTblPolicyIDCol = "policy_id"
 	PolicyDepTblDepIDCol    = "dependency_id"
+
+	SchemaTbl              = "schema"
+	SchemaTblIDCol         = "id"
+	SchemaTblDefinitionCol = "definition"
+	SchemaDefaultID        = 1
 )
 
+type Schema struct {
+	ID          uint
+	Description string
+	Disabled    bool
+	Definition  SchemaDefWrapper
+}
+
+type SchemaDefWrapper struct {
+	*schemav1.Schema
+}
+
+func (sdw SchemaDefWrapper) Value() (driver.Value, error) {
+	return sdw.Schema.MarshalVT()
+}
+
+func (sdw *SchemaDefWrapper) Scan(src interface{}) error {
+	var source []byte
+	switch t := src.(type) {
+	case nil:
+		return nil
+	case string:
+		source = []byte(t)
+	case []byte:
+		source = t
+	default:
+		return fmt.Errorf("unexpected type for schema definition: %T", src)
+	}
+
+	sdw.Schema = &schemav1.Schema{}
+	if err := sdw.Schema.UnmarshalVT(source); err != nil {
+		return fmt.Errorf("failed to unmarshal schema definition: %w", err)
+	}
+
+	return nil
+}
```

Comment:
I think we should store the schema as JSON using the database's native JSON type. We have a request to do the same for policies as well but that requires a migration step that we can't do at the moment. However, because this is a new table, we can store the data as JSON from the start.